src/main/ts/wallet/MoneroWalletRpc.ts

```diff
@@ -2507,6 +2507,7 @@ class WalletPoller {
       } catch (err: any) {
         that.numPolling--;
         console.error("Failed to background poll " + await that.wallet.getPath());
+        console.error(err);
```

Comment:
This error is "expected" when the wallet disconnects, so a full stacktrace pointing to the polling internals may be too verbose.

However, I added the error message to the logs in my commit to catch and print all errors invoking listeners: https://github.com/woodser/monero-ts/commit/ce1589784284372b46d7d5cb0abda89e9119f834



This is an expected error when the wallet disconnects, so a full stacktrace pointing to the polling internals may be too verbose. However, I added a log with the error message:

https://github.com/woodser/monero-ts/commit/ce1589784284372b46d7d5cb0abda89e9119f834